Symfony/CS/Fixer/PSR2/VisibilityFixer.php

```diff
@@ -61,9 +61,9 @@ public function getDescription()
      *
      * Token at given index is prepended by attributes.
      *
-     * @param Tokens $tokens  Tokens collection
-     * @param int    $index   token index
-     * @param array  $attribs array of token attributes
+     * @param Tokens                    $tokens  Tokens collection
+     * @param int                       $index   token index
+     * @param array<string, Token|null> $attribs array of token attributes
```

Comment:
> > Are you sure this is supported by all IDEs?
>
> ;-)

Don't follow IDE notation, follow standardized notation!

[A short and incomplete list of IDEs](https://en.wikipedia.org/wiki/Comparison_of_integrated_development_environments#PHP) you may check. I could blindly bet - no, not supported by all IDEs, especially when not IDEs support PHPDoc itself.